.github/workflows/ci.yml

```diff
@@ -58,8 +60,10 @@ jobs:
       - name: Set up locales
         run: ./hack/setup-locales.sh
 
-      - name: Download dependencies
-        run: composer install --classmap-authoritative
+      - uses: "ramsey/composer-install@v3"
+        with:
+          # We need to ignore PHP because Psalm blocks installing it
```

Comment:
I cannot merge with this being part of the PR.

Reply:
So what do you suggest?

Reply:
We can remove psalm + the plugin before running tests on PHP 8.4. There is no point for the dependency in that job anyway.

Reply:
I suggest we wait until [they resolve the issue](https://github.com/vimeo/psalm/pull/10928). I subscribe for that PR. Once support is there, we can move forward here.

Reply:
But this is not a blocker for PHP 8.4. Psalm is only executed on a separate job that runs with PHP 8.1:
https://github.com/moneyphp/money/blob/76a94ba21bd649adbcbe3d5b0646cadb8da3d458/.github/workflows/ci.yml#L78

Reply:
First, it's not a bad idea to change that into the latest stable version (8.3). Secondly, since Psalm uses the strategy as this package, we cannot install dependencies, which means we have to wait for them to solve the issue. I don't see a problem here. We are not even in the RC phase of PHP8.4. We wait.

Reply:
> We can remove psalm + the plugin before running tests on PHP 8.4. There is no point for the dependency in that job anyway.

Let's go for this one then.